.github/workflows/reusable_packaging_and_deployment.yml

```diff
@@ -40,10 +52,16 @@ jobs:
           CC: ${{ steps.install_cc.outputs.cc }}
           CXX: ${{ steps.install_cc.outputs.cxx }}
         run: |
-          pip3 install "conan==1.59" ${{ fromJSON('["", "gcovr"]')[inputs.with-coverage] }}
+          pip3 install "conan==1.59" yq
```

Comment:
I would suggest `conan==1.60.1`. With 1.60 support for gcc13 was added (see https://docs.conan.io/1/changelog.html#id3)

This would make yq obsolete

Reply:
Ok, will change didn't notice conan 1.60 was a thing